File: Log de Programação/2°Bi/Lista 2/SalaPy/Lista2-sala_Exercício 6 - Equação.py
import math
A = float(input("Insira A: "))
B = float(input("Insira B: "))
C = float(input("Insira C: "))
if (A == 0):
    print("Essa equação não é do 2° grau!")
else:
    D = (pow(B,2)) - 4 * A * C
    if (D < 0):
        print("Não há raízes reais para essa equação")
    else:
        x1 = (- B + math.sqrt(D)) / (2 * A) 
        x2 = (- B - math.sqrt(D)) / (2 * A)
        print("As raízes dessa equação são {:.2f} e {:.2f}".format(x1, x2))

File: 1 ano/Log de Programação/2°Bi/Lista 2/SalaC/Lista2-sala_Exercício 6 - Equação.c
#include <stdio.h>
#include <math.h>

int main()
{
    float a, b, c, d, x1, x2;

    printf("Insira A: ");
    scanf("%f", &a);
    printf("Insira B: ");
    scanf("%f", &b);
    printf("Insira C: ");
    scanf("%f", &c);

    if (a == 0){
        printf("Esta equacao nao eh do 2o grau");
    }
    else{
        d = pow(b,2) - (4 * a * c);
        if (d < 0){
            printf("Esta equacao sao nao tem raizes reais");
        }
        else{
            x1 = (-b + sqrt(d)) / (2 * a);
            x2 = (-b - sqrt(d)) / (2 * a);
            printf("As raizes dessa equacao sao: %.2f e %.2f", x1, x2);
        }
    }
}